### DISCUSSION TITLE
Adding additional Webhooks to discord loot logger.

### DISCUSSION THREAD

**Discussion by Bloodrein** [runelite/runelite#15291] [Q&A] [answered]
Currently the discord loot logger plugin can only support one web-hook at a time specific to one server, would it be possible to add an additional URL field for multiple server web-hooks?

![2022-07-07_11-06-16](https://user-images.githubusercontent.com/108894824/177840319-d3b96b60-f9e9-469f-8a05-8d7fafdb1186.jpg)

**Comment by Enriath** [comment, marked as answer]
Issues and suggestions for hub plugins must be made on that plugin's repository; they are not maintained by us. The repo is usually accessible from the Support link on the plugin.
